docs/src/rsh/errors/index.md

```diff
@@ -2566,6 +2566,12 @@ const x = forall(UInt);
 assert(x >= 0);
 ```
 
+## {#RX0001} RX0001
+
+This error should never be encountered.
+If you receive this error, please report it to the Reach team along with the program that produced it.
```

Comment:
Then why isn't it an `impossible`?

Reply:
I piggy-backed off how DLE_Impossible uses `expect_thrown` in the connectors, but I'll special case this to use `impossible`